project-keeper/src/main/java/com/exasol/projectkeeper/validators/pom/PomFileGenerator.java

```diff
@@ -42,6 +42,7 @@ public class PomFileGenerator {
             new SimplePluginTemplateGenerator("maven_templates/maven-javadoc-plugin.xml", MAVEN_CENTRAL),
             new SimplePluginTemplateGenerator("maven_templates/nexus-staging-maven-plugin.xml", MAVEN_CENTRAL),
             new SimplePluginTemplateGenerator("maven_templates/maven-dependency-plugin.xml", UDF_COVERAGE),
+            new SimplePluginTemplateGenerator("maven_templates/native-image-maven-plugin.xml", NATIVE_IMAGE),
```

Comment:
When a change like this only requires adding a single line means the architecture is good 👍